twake/frontend/src/app/components/search-popup/tabs/drive.tsx

```diff
@@ -0,0 +1,43 @@
+import * as Text from '@atoms/text';
+import Languages from 'app/features/global/services/languages-service';
+import { useSearchDriveItems } from 'app/features/search/hooks/use-search-drive-items';
+import { SearchInputState } from 'app/features/search/state/search-input';
+import { useRecoilValue } from 'recoil';
+import FileResult from '../parts/file-result';
+import NothingFound from '../parts/nothing-found';
+
+export default () => {
+  const input = useRecoilValue(SearchInputState);
+  const isRecent = input?.query?.trim()?.length === 0;
+  const { driveItems, loading } = useSearchDriveItems();
+
+  if (driveItems.length === 0 && !loading) return <NothingFound />;
+
+  return (
+    <div>
+      {!!isRecent && (
+        <Text.Subtitle className="block">
+          {Languages.t('components.searchpopup.recent_files')}
+        </Text.Subtitle>
+      )}
+
+      <div className={'-mx-2'}>
+        <DriveItemsResults />
+      </div>
+    </div>
+  );
+};
+
+export const DriveItemsResults = (props: { max?: number }) => {
+  const { driveItems, loading } = useSearchDriveItems();
+
+  if (driveItems.length === 0 && !loading) return <NothingFound />;
+
+  return (
+    <>
+      {driveItems.slice(0, props?.max || driveItems.length).map(item => (
+        <FileResult key={item.id} file={item} />
```

Comment:
Does this work when clicking on it ? Does it redirect to the Drive correct location ?

Reply:
I just updated the entire component for displaying the result item and the download button is working now. 

<img width="1399" alt="Screenshot 2023-02-21 at 10 39 40 AM" src="https://user-images.githubusercontent.com/7594703/220307395-331e21a6-5c4f-40f7-9a90-07937c5bf3e3.png">